server/tests/audit_math/test_bravo.py

```diff
@@ -598,3 +672,53 @@ def test_tied_contest():
         "all-ballots": {"type": "all-ballots", "size": 4992420, "prob": None},
     },
 }
+
+
+round2_sample_sizes = {
+    "test1": {
+        "asn": {"type": "ASN", "size": -12, "prob": 0.0},
```

Comment:
what's with all these negative sample sizes?

Reply:
It's a quirk of the math. I guess we could be rounding them to zero, but these are cases that should never really be seen by Arlo....